cegs_portal/search/views/v1/reg_effects.py

```diff
@@ -231,8 +239,9 @@ class TargetEffectsView(FeatureEffectsView):
     table_partial = "search/v1/partials/_target_reg_effect.html"
     tsv_renderer = target_data
 
-    def get_data(self, options, feature_id) -> tuple[QuerySet[RegulatoryEffectObservation], DNAFeature]:
+    def get_data(self, options, feature_id) -> tuple[QuerySet[RegulatoryEffectObservation], DNAFeature, DNAFeature]:
         feature = RegEffectSearch.id_feature_search(feature_id)
+        dna_feature = DNAFeature.objects.get(accession_id=feature_id)
```

Comment:
feature and dna_feature are the same thing.

```
    @classmethod
    def id_feature_search(cls, feature_accession_id: str):
        return DNAFeature.objects.get(accession_id=feature_accession_id)
```

You don't need to do a separate lookup.